Documentation/devel/on-disk-format.md

```diff
@@ -1,31 +1,25 @@
 On disk format
 ==============
 
-The data directory is normally `/var/lib/rkt` but rkt can be requested to use
-another directory via the `--dir` command line argument.
+The data directory is normally `/var/lib/rkt` but rkt can be requested to use another directory via the `--dir` command line argument.
```

Comment:
The data directory is `/var/lib/rkt` by default. The `--dir` command line option can be used to change this location.